src/riak_dt_orswot.erl

```diff
@@ -118,37 +120,45 @@
 
 -type precondition_error() :: {error, {precondition ,{not_present, member()}}}.
 
+-define(EMPTY_ORSWOT, {[],#{},[]}).
+
 -spec new() -> orswot().
 new() ->
-    {riak_dt_vclock:fresh(), orddict:new(), orddict:new()}.
+    {riak_dt_vclock:fresh(), maps:new(), orddict:new()}.
 
 %% @doc sets the clock in the Set to that `Clock'. Used by a
 %% containing Map for sub-CRDTs
 -spec parent_clock(riak_dt_vclock:vclock(), orswot()) -> orswot().
 parent_clock(Clock, {_SetClock, Entries, Deferred}) ->
     {Clock, Entries, Deferred}.
 
--spec value(orswot()) -> [member()].
+-spec value(orswot() | legacy_orswot()) -> [member()].
+value({_Clock, Entries, _Deferred}) when is_list(Entries) ->
+    [K || {K, _Dots} <- orddict:to_list(Entries)];
```

Comment:
Yeah, but in this case you are checking is_list(Entries) then why orddict:to_list?